src/Tools/DynamoShapeManager/Utilities.cs

```diff
@@ -104,8 +104,8 @@ public static void PreloadAsmFromPath(string preloaderLocation, string asmLocati
 
             var preloaderPath = Path.Combine(preloaderLocation, PreloaderAssembly);
 
-            Debug.WriteLine("ASM Preloader: {0}", preloaderPath);
-            Debug.WriteLine("ASM Location: {0}", asmLocation);
+            Debug.WriteLine(string.Format("ASM Preloader: {0}", preloaderPath));
+            Debug.WriteLine(string.Format("ASM Location: {0}", asmLocation));
```

Comment:
The current print out is `ASM Location: {0}` on the console, so a `string.Format` is required.